### ISSUE TITLE
cleanup_dead_jobs() , active jobs, empty case not handled 

### ISSUE THREAD

**Issue by upagrah** [wooey/Wooey#261]
Hello. I am new to open source software. While i was implementing your wooey software for a project, I found out that the active tasks returned by celery is some times empty and the iteritems is not able to handle an empty dict perhaps. I handled it as follows. (File : Wooey/tasks.py)
```python
#ORIGINALLY IN TASKS
inspect = celery_app.control.inspect()
    active_tasks = {task['id'] for worker, tasks in six.iteritems(inspect.active()) for task in tasks}
#MODIFIED TASKS
    active_tasks={}
    if(inspect.active()):
        active_tasks = {task['id'] for worker, tasks in six.iteritems(inspect.active()) for task in tasks}
```
I am sorry as I am not yet aware of how to submit modifications or proposals to opensource project. If any one of you can guide me to any link , which shows how can i propose changes on github opensource projects,i will try and add this.Thank you.
Handling that way, is it correct? May be , may be not. Hence posted in issue.

**Comment by Chris7** [comment]
Hi @upagrah,

Thanks for the report. an empty dict should be fine. If it's `None`, or another type, it might be a problem. What version of celery are you using, and could you provide the stacktrace that is indicating this error (or just log the value of `inspect.active()`)?

**Comment by smallzhao** [comment]
I also have this problem, my celery version is in accordance with requirements.txt
May be caused by the following function returning None
(File:celery.app.control.Inspect) 
```    python
    def _prepare(self, reply):
        if reply:
            by_node = flatten_reply(reply)
            if (self.destination and
                    not isinstance(self.destination, (list, tuple))):
                return by_node.get(self.destination)  # this mabe return a None
            if self.pattern:
                pattern = self.pattern
                matcher = self.matcher
                return {node: reply for node, reply in items(by_node)
                        if match(node, pattern, matcher)}
            return by_node
```

**Comment by Chris7** [comment]
Ah, that makes sense. I'll try to get to this when I have some time, but feel free to beat me to it with a PR :)

**Comment by Chris7** [comment]
Fixed via #281
